docs/users/exchanges.md

```diff
@@ -0,0 +1,361 @@
+```
+3. User does a L2 transfer from his account to the destination account using the web wallet
+```js
+  const infoAccountUser = (await hermez.CoordinatorAPI.getAccounts(hermezUserWallet.hermezEthereumAddress, [tokenERC20.id]))
+    .accounts[0];
+
+  const state = await hermez.CoordinatorAPI.getState();
+  const usdTokenExchangeRate = tokenERC20.USD;
+  const fee = usdTokenExchangeRate ? state.recommendedFee.createAccountInternal / usdTokenExchangeRate : 0;
+
+  // user creates transaction to deposit 10 ether into exchange account
+  // deposit 10 ether
+  const userDepositToExchange = hermez.Utils.getTokenAmountBigInt("10.0", 18);
+  const compressedUserDepositToExchange = hermez.HermezCompressedAmount.compressAmount(userDepositToExchange);
+  // the following transaction would:
+  // - create an account for the exchange in hermez network
+  // - transfer to exchange account 0.1 eth
+  const transferToExchange = {
+    from: infoAccountUser.accountIndex,
+    to: hermezExchangeUserWallet.publicKeyBase64,
+    amount: compressedUserDepositToExchange,
+    fee : fee
+  };
+  console.log("transferToExchange: ", transferToExchange, fee);
+  // send tx to hermez network
+  await hermez.Tx.generateAndSendL2Tx(transferToExchange, hermezUserWallet, tokenERC20);
+```
+4. Exchange monitors balance of the account `L2ExUser-ETH`, and once transfer is complete, exchange performs transfer from `L2ExUser-ETH` to `L2Ex-ETH` for 10 ETH.
+```js
+  const pollingExchangeAddr = true;
+  while (pollingExchangeAddr){
+    const accountExchangeInfo = await hermez.CoordinatorAPI.getAccounts(hermezExchangeUserWallet.publicKeyBase64, [tokenERC20.id]);
+    if (accountExchangeInfo.accounts.length === 0){
+      console.log("Waiting for user deposit to be forged...");
+      await sleep(10000);
+    } else {
+      console.log("<=== Received deposit from user ===>");
+      console.log(`accountExchangeInfo:\n ${accountExchangeInfo.accounts[0]}`);
+      break;
+    }
+  }
+
+  const infoAccountExchangeUser = (await hermez.CoordinatorAPI.getAccounts(hermezExchangeUserWallet.publicKeyBase64, [tokenERC20.id]))
+   .accounts[0];
+
+
+  // Transfer funds to main exchange account
+  // generate L2 transaction
+  const l2TxTransfer = {
+    from: infoAccountExchangeUser.accountIndex,
+    to:  infoAccountExchange.accountIndex,
+    amount: compressedUserDepositToExchange,
+    fee: fee
+  };
+
+  const transferResponse = await hermez.Tx.generateAndSendL2Tx(l2TxTransfer, hermezExchangeUserWallet, tokenERC20).catch(console.log);
+  console.log("transferResponse: ", transferResponse);
+```
+
+### Full Example
+```js
+const hermez = require("@hermeznetwork/hermezjs");
+
+const EXAMPLES_HERMEZ_API_URL = "https://api.testnet.hermez.io/";
+const EXAMPLES_HERMEZ_ROLLUP_ADDRESS = "0x14a3b6f3328766c7421034e14472f5c14c5ba090";
+const EXAMPLES_HERMEZ_WDELAYER_ADDRESS = "0x6ea0abf3ef52d24427043cad3ec26aa4f2c8e8fd";
+
+// Provide your own values
+const EXAMPLES_WEB3_URL = "http://----";
```

Comment:
In section `pre-requisites`, those variables are empty.
I would either change `pre-requisites` section or this one in order to match them